release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3920,4 +3920,23 @@ $ oc adm release info 4.12.27 --pullspecs
 [id="ocp-4-12-27-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-28"]
+=== RHBA-2023:4440 - {product-title} 4.12.28 bug fix update
+
+Issued: 2023-08-09
+
+{product-title} release 4.12.28 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:4440[RHBA-2023:4440] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4443[RHBA-2023:4443] advisory.
```

Comment:
Everything looks good, but these two advisory links are giving me 404 errors. If it's because they won't be publicly available until the 9th, disregard this comment.